Lesson16/main.js

```diff
@@ -0,0 +1,87 @@
+const tabContentList = document.getElementById("js-tab-content");
+const imgWrapper = document.getElementById("js-tab-img");
+const tabs = document.getElementsByClassName("tab-menu__item");
+
+
+async function getJsonOrError() {
+  const response = await fetch("./data.json");
+  const json = await fetchErrorHandling(response);
+  return json;
+}
+
+async function fetchErrorHandling(response) {
+  if (response.ok) {
+    return await response.json();
+  } else {
+    throw new Error("サーバーエラーが発生しました");
+  }
+}
+
+async function tryGetData() {
+  try {
+    const data = await getJsonOrError();
+    return data;
+  } catch (error) {
+    tabContentList.replaceWith(error);
+  }
+}
+
+async function createElement({ article }) {
+  const articleLength = 4;
```

Comment:
IMO
```suggestion
  const NumberOfArticlesToDisplay = 4;
```
I think the purpose and name of this variable are different.
For example
```
Number of articles to show
Number of articles to display
```
I don't speak English very well, but ideally I should be able to understand a job just by looking at it!

Reply:
Thanks for the review:)

I thought you were right.
If I use articleLength, it's just the number of articles, and it would mean the same thing as article.length...

Also, as Morita-sensei mentioned in the discord, I changed the number of articles to be set by article.length.

Therefore, I have deleted the part you pointed out.

But I learned a lot from the review.
I'll keep it for future reference.
f2374d3